benchmarks/test_pkresponse.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+import os
+from pyccl.pkresponse import Pmm_resp, darkemu_Pgm_resp, darkemu_Pgg_resp
+import pyccl as ccl
+
+data_directory_path = os.path.expanduser("benchmarks/data/SSC-Terasawa24/")
+
+# Set cosmology
+Om = 0.3156
+ob = 0.02225
+oc = 0.1198
+OL = 0.6844
+As = 2.2065e-9
+ns = 0.9645
+h = 0.6727
+
+cosmo = ccl.Cosmology(
+    Omega_c=oc / (h**2),
+    Omega_b=ob / (h**2),
+    h=h,
+    n_s=ns,
+    A_s=As,
+    m_nu=0.06,
+    transfer_function="boltzmann_camb",
+    extra_parameters={"camb": {"halofit_version": "takahashi"}},
+)
+
+
+# Construct the full path for each data file (z=0)
+k_data_path = os.path.join(data_directory_path, "k_h.npy")
+k_data_mm_path = os.path.join(data_directory_path, "k_h_mm.npy")
+Pmm_resp_data_path = os.path.join(data_directory_path, "Pmm_resp_z0.npy")
```

Comment:
You have so much more data generated, at different redshifts. Why don't you use it?

Reply:
I added the tests for higher redshifts, z ~ [0.5, 1.0, 1.5].